src/app/static/src/tests/integration/projects.itest.ts

```diff
@@ -204,9 +204,9 @@ describe("Projects actions", () => {
 
         await actions.renameProject({commit, dispatch, state, rootState} as any, projectPayload);
         setTimeout(() => {
-            expect(commit.mock.calls[2][0]).toStrictEqual({type: ProjectsMutations.ClearCurrentVersion});
-            expect(commit.mock.calls.length).toBe(3);
-            expect(dispatch.mock.calls[0][0]).toBe("getProjects");
+            expect(commit.mock.calls.length).toBe(2);
```

Comment:
Once I added the integration tests to the build, this one was failing.

Reply:
Makes sense, corresponds to a recent change